misk/web/tabs/dashboard/src/components/SidebarComponent.tsx

```diff
@@ -1,35 +1,24 @@
 import { Menu, MenuItem } from "@blueprintjs/core"
+import { IMiskAdminTabs } from "@misk/common"
 import * as React from "react"
 import styled from "styled-components" 
-import { IMenuItem } from "../utils/menu"
 
 interface ISidebarProps {
-  menuItems: IMenuItem[]
+  adminTabs: IMiskAdminTabs
 }
 
 const Sidebar = styled.div`
   position: absolute;
 `
 
-export class SidebarComponent extends React.Component<ISidebarProps, {}> {
-  private renderedMenuItems: any
+const buildMenuItems = (adminTabs: IMiskAdminTabs) => (
+  Object.entries(adminTabs).map((tab) => <MenuItem key={tab[0]} href={tab[1].url_path_prefix} className="" icon={tab[1].icon} text={tab[1].name}/>)
```

Comment:
Can we use array destructuring to name `tab[0]` and `tab[1]` here? It could help some know what those objects are supposed to be:

```
  Object.entries(adminTabs).map(([tabName, tabConfig]) => {
    <MenuItem key={tabName} href={tabConfig.url_path_prefix} className="" icon={tabConfig.icon} text={tabConfig.name}/>)
  }
```

Reply:
Really like this, fixed it in Loader tab as well!